homeassistant/components/scene/hunterdouglas_powerview.py

```diff
@@ -12,15 +12,15 @@
 _LOGGER = logging.getLogger(__name__)
 REQUIREMENTS = [
     'https://github.com/sander76/powerviewApi/'
-    'archive/cc6f75dd39160d4aaf46cb2ed9220136b924bcb4.zip#powerviewApi==0.2']
+    'archive/master.zip#powerviewApi==0.3.15']
```

Comment:
The requirement must point to a commit to avoid the issue which needed to be addressed two days ago.